grails-app/views/occurrence/_map.gsp

```diff
@@ -399,7 +399,7 @@ a.colour-by-legend-toggle {
     }
 </r:script>
 
-<div class="hide">
+<div style="display: none;">
```

Comment:
https://v4-alpha.getbootstrap.com/migration/
Explainbed under "Responsive utilities". I'll try to sort them out with a custom hide class.